server/routes/clientRouter.js

```diff
@@ -2,9 +2,10 @@ import express from "express"
 
 const router = new express.Router()
 
-const clientRoutes = []
+const clientRoutes = ["/", "/pets"]
+
```

Comment:
We can actually remove `"/"` path after we update the rootRouter to handle a redirect